site/src/components/keyboard-controls/KeyboardControl.tsx

```diff
@@ -0,0 +1,56 @@
+import { FC, ReactNode, Fragment } from "react";
```

Comment:
Should this be in `site/src/components/components/keyboard-controls/KeyboardControl.tsx` as it's specific to the components page?

Reply:
I figured it could theoretically be used anywhere. There's nothing about its implementation that restricts it to component pages.

TBH, I think that whole components directory could do with a re-org. I've been thinking it might make sense to organise the components into: 

- components that are used inside the content of a page (e.g. KeyboardControls, LivePreview, Card, etc.)
- components that are used in the layouts _around_ the page content (e.g. Header, Footer, etc.)
- Maybe, there's a 3rd "special snowflake" category for stuff really is unique to a certain page (e.g. some of the stuff on our homepage)

Perhaps I'll raise an issue to discuss that and, if needed, move things around in a separate PR.

If it's Ok with you though, can I leave KeyboardControl where it is for the time being?

Reply:
yeah that sounds good :)